core/actuator.py

```diff
@@ -17,81 +17,95 @@
 Classes: Actuator
 """
 from abc import ABC, abstractmethod
+from typing import Any, Callable, Optional
 import logging
-from core.utils import set_log_level
+from core import utils
+from core import connection
 
 class Actuator(ABC):
-    """Class from which all actuator capabilities must inherit. Is assumes there
-    is a "Topic" param and automatically registers to subscribe to that topic
-    with all of the passed in connections. A default implementation is provided
-    for all but the on_message method which must be overridden.
+    """ Class from which all actuator capabilities must inherit. Is assumes there
```

Comment:
> Class from which all actuator capabilities must inherit. Is assumes there

Class from which all actuator capabilities must inherit. **It** assumes there

I know this type was in the original but I'm just now seeing it.